features/workflow.feature

```diff
@@ -16,125 +16,122 @@ Feature: Workflow
         - Workflows must be one-directional (no loops)
         - All subsequent jobs must checkout from the same SHA as the previous job
 
-    Scenario: Failure
-        Given an existing pipeline with the workflow:
-            | job | requires |
-            | FOO | ~commit  |
-            | BAR | FOO      |
-        When the "FOO" job is started
-        And the build failed
-        Then the "BAR" job is not started
-
     Scenario: Serially
-        Given an existing pipeline with the workflow:
-            | job | requires |
-            | FOO | ~commit  |
-            | BAR | FOO      |
-        When the "FOO" job is started
-        And the build succeeded
-        Then the "BAR" job is started
-        And that "BAR" build uses the same SHA as the "FOO" build
+        Given an existing pipeline on "serially" branch with the workflow jobs:
+            | job       | requires  |
+            | SIMPLE    | ~commit   |
+            | PARALLEL1 | SIMPLE    |
```

Comment:
I feed that `PARALLEL1` is not suitable for job name.

Reply:
I assume to use [this repo](https://github.com/screwdriver-cd-test/functional-workflow/pull/1), so I would like to use `PARALLEL1` here.